spec/models/project_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Project, type: :model do
+  context "invalid params" do
```

Comment:
`context` description should start with word `with` or `when`.

http://betterspecs.org/#contexts